uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/ClientAdminEndpointsIntegrationTests.java

```diff
@@ -170,6 +171,21 @@ public void createClientWithSecondarySecret() {
         assertEquals(HttpStatus.CREATED, result.getStatusCode());
     }
 
+    @Test
+    public void createClientWithEmptySecret() {
```

Comment:
I always do a squash commit because this is my way the last years... I do many commits. but if a featue/fix is ready, then I squash it into one commit and merge to repo only the consolidated version, .... I can squash everything now if you dont like the small commits